src/screens/TripDetailsScreen.tsx

```
@@ -91,16 +143,83 @@
+                <ul>
+                  {Array.from(payment.shares).map(([user, amount]) => (
+                    <li key={user}>
+                      {user}: {amount.toFixed(2)}
+                    </li>
+                  ))}
+                </ul>
+              </div>
+              <button
+                className="button is-warning is-light"
+                onClick={() => handleEditPayment(payment)}
+              >
```

Biome (1.9.4)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)